src/main/java/com/ericsson/ei/controller/FailedNotificationControllerImpl.java

```diff
@@ -35,29 +30,33 @@
 import com.ericsson.ei.utils.ResponseMessage;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
 /**
- * This class represents the REST GET mechanism to extract the aggregated data on the basis of the SubscriptionName from the Missed Notification
- * Object.
+ * This class contains logic for retrieving failed notifications for the given
+ * subscription.
  */
 @Component
 @CrossOrigin
-@Api(value = "queryMissedNotification", tags = {"Missed notifications"})
-public class QueryMissedNotificationControllerImpl implements QueryMissedNotificationController {
+@Api(value = "failedNotifications", tags = { "Failed notifications" })
+public class FailedNotificationControllerImpl implements FailedNotificationController {
 
-    private final static Logger LOGGER = LoggerFactory.getLogger(QueryMissedNotificationControllerImpl.class);
+    private final static Logger LOGGER = LoggerFactory.getLogger(FailedNotificationControllerImpl.class);
 
     @Autowired
     private ProcessMissedNotification processMissedNotification;
 
     /**
-     * This method is responsible for the REST GET mechanism to extract the data on the basis of the SubscriptionName from the Missed Notification
-     * Object.
+     * This method is responsible for the REST GET mechanism to extract the data on
+     * the basis of the SubscriptionName from the Failed Notification Object.
```

Comment:
The parameter is now subscriptionName, so if this javadoc refers directly to the paramater it should be a lower case s.

Reply:
done